specs/gpg/generate_gpg_key.yaml

```diff
@@ -0,0 +1,11 @@
+---
+name: Generate a new GPG key
+command: "gpg --full-generate-key"
+tags:
+  - gpg
+description: Interactively generate a new GPG key.
+author: Chay Choong
+author_url: "https://github.com/chaychoong"
+shells:
+  - Zsh
```

Comment:
One small thing I'm noticing here is that these specs are only scoped to to Bash and ZSH. Since commands don't use any shell-specific syntax, they can work for any shell.

The easiest fix for this would be to just remove the `shells` field entirely, since it's optional